sql/src/main/java/org/apache/druid/sql/calcite/planner/DruidOperatorTable.java

```diff
@@ -340,6 +340,7 @@ public class DruidOperatorTable implements SqlOperatorTable
                    .add(new AliasedOperatorConversion(CHARACTER_LENGTH_CONVERSION, "LENGTH"))
                    .add(new AliasedOperatorConversion(CHARACTER_LENGTH_CONVERSION, "STRLEN"))
                    .add(new DirectOperatorConversion(SqlStdOperatorTable.CONCAT, "concat"))
+                   .add(new DirectOperatorConversion(SqlStdOperatorTable.DESC, "desc"))
```

Comment:
a `DirectOperatorConversion` doesn't really seem quite correct since it is for things that have a direct native expression to translate into. It looks like the same effect can be had by instead adding this to `DruidConvertTable.STANDARD_CONVERTLET_OPERATORS` which just allows it to pass through to see if anything else handles it (at least all the tests pass), though i'm not really sure if that is the most appropriate way to do this either since I guess this does have a window operator associated with it, but it does seem better than using a `DirectOperatorConversion`.